dev-python/boost-histogram/boost-histogram-1.4.0.ebuild

```diff
@@ -0,0 +1,42 @@
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+DISTUTILS_USE_PEP517=setuptools
+inherit distutils-r1 pypi
+
+DESCRIPTION="Python bindings for the C++14 Boost::Histogram library"
+HOMEPAGE="https://github.com/scikit-hep/boost-histogram"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	dev-libs/boost[python,${PYTHON_USEDEP}]
+	dev-python/numpy[${PYTHON_USEDEP}]
+	test? (
```

Comment:
test dependencies should be BDEPEND not RDEPEND

Reply:
I guess i also do not need pytest explicitly if i declare pytest via `enable_tests`?

Reply:
> I guess i also do not need pytest explicitly if i declare pytest via `enable_tests`?

Yes, it is indeed added automatically.